pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -186,6 +186,11 @@ message BrokerEntryMetadata {
     optional uint64 index = 2;
 }
 
+message ManagedLedgerInfoMetadata {
```

Comment:
@gaoran10 We should move it to `MLDataFormats.proto`? The PulsarApi.proto is used for the broker and client interaction.

Reply:
Ok, I'll move it.